### DISCUSSION TITLE
请教merlin系统安装alist问题

### DISCUSSION THREAD

**Discussion by hubgha-hub** [AlistGo/alist#6210] [Q&A]
ssh终端连接后使用alist server命令可以运行,关掉终端后服务也随之关闭了,怎样才能后台运行不掉,谢谢!

**Comment by anwen-anyi** [comment]
merlin是啥系统？ 你可以参考Linux或者Windows那种 使用脚本/守护进程方式启动

**Reply by hubgha-hub** [reply]
华硕路由器梅林改版系统https://www.asuswrt-merlin.net/ ,arm架构

**Reply by anwen-anyi** [reply]
> 华硕路由器梅林改版系统https://www.asuswrt-merlin.net/ ,arm架构

你试试这个呢？
Alist 梅林-插件仓库：https://github.com/everstu/Koolcenter_alist
